rubyfmt.rb

```diff
@@ -205,8 +204,37 @@ def on_regexp_literal(*args)
 end
 
 GC.disable
-file_data = File.read(ARGV[0])
-parsed = Parser.new(file_data).parse
-STDERR.puts(parsed.inspect) unless ENV["RUBYFMT_USE_RELEASE"]
-Rubyfmt::format_to_stdout(file_data, parsed)
-STDOUT.close
+
+def parse_file(fn)
+  file_data = File.read(fn)
+  [file_data, Parser.new(file_data).parse]
+end
+
+def rubyfmt_dir(dn)
+  files = Dir[File.join(dn, "**/*.rb")]
```

Comment:
you may like to also find Gemfile, Capfile, .gemspec, *.rake, config.ru, etc

Reply:
also files with no extension but with ruby shebang (SORRY)

Reply:
If you'd like, I'd welcome a patch that supports this usecase, but for right now I'm going to merge this.